raster/r.univar/r.univar_main.c

```diff
@@ -165,8 +165,8 @@ int main(int argc, char *argv[])
     /* table field separator */
     zone_info.sep = G_option_to_separator(param.separator);
 
-    zone_info.min = 0.0 / 0.0;  /* set to nan as default */
-    zone_info.max = 0.0 / 0.0;  /* set to nan as default */
+    zone_info.min = NAN;
+    zone_info.max = NAN;
```

Comment:
```suggestion
    zone_info.min = 0;
    zone_info.max = 0;
```

`zone_info.min` and `zone_info.max` are integers and there is no NaN for integers, only floats.
The previous solution `0.0 / 0.0;` was casted to int, on my machine it turns out to `458864`. Better with `0`.